### DISCUSSION TITLE
GitHub Classroom Survey Problem

### DISCUSSION THREAD

**Discussion by sjengle** [community/community#31067] [GitHub Education] [answered]
In GitHub Classroom, there is a blue prompt that says:

> We value your feedback! Please take a few minutes to complete our [survey](https://aka.ms/classroom-survey).

There is a question on that survey that says:

> What new capabilities would you like to see for Classroom? Select up to 3

It will not let me submit with 3 items selected. I was only have to submit with exactly 1 item selected.

**Comment by melvincwng** [comment]
@sjengle Hi Sophie, for bugs related to GitHub's product, kindly raise a ticket to [GitHub Bug Report](https://support.github.com/contact/bug-report).

Hope this resolves and answers your issue/question 😊

**Comment by RyanHecht** [comment, marked as answer]
Hi there @sjengle, good catch! We've fixed this :) Thanks for the callout!

**Reply by DMGregory-Sheridan** [reply]
A similar problem turned up again in this year's survey (2024-25). The question "Do you feel you have access to adequate support when facing coding challenges? Select all that apply." allows only one selection.
